src/VirtoCommerce.CustomerModule.Web/Scripts/blades/customer-detail.js

```diff
@@ -1,69 +1,88 @@
-        blade.title = 'customer.blades.contact-detail.title-new';
-        blade.currentEntity = angular.extend({
-            organizations: []
-        }, blade.currentEntity);
+            if (blade.isNew) {
+                blade.title = 'customer.blades.contact-detail.title-new';
+                blade.currentEntity = angular.extend({
+                    organizations: []
+                }, blade.currentEntity);
 
-        if (blade.parentBlade.currentEntity.id) {
-            blade.currentEntity.organizations.push(blade.parentBlade.currentEntity.id);
-        }
+                if (blade.parentBlade.currentEntity.id) {
+                    blade.currentEntity.organizations.push(blade.parentBlade.currentEntity.id);
+                }
 
-        blade.fillDynamicProperties();
-    } else {
-        blade.subtitle = 'customer.blades.contact-detail.subtitle';
-    }
+                blade.fillDynamicProperties();
+            } else {
+                blade.subtitle = 'customer.blades.contact-detail.subtitle';
+            }
 
-    // datepicker
-    $scope.datepickers = {};
-    $scope.today = new Date();
+            // datepicker
+            $scope.datepickers = {};
+            $scope.today = new Date();
 
-    $scope.open = function ($event, which) {
-        $event.preventDefault();
-        $event.stopPropagation();
+            $scope.open = function ($event, which) {
+                $event.preventDefault();
+                $event.stopPropagation();
 
-        $scope.datepickers[which] = true;
-    };
+                $scope.datepickers[which] = true;
+            };
 
-    $scope.timeZones = timeZones.query();
-    $scope.groups = settings.getValues({ id: 'Customer.MemberGroups' });
+            $scope.timeZones = timeZones.query();
+            $scope.groups = settings.getValues({ id: 'Customer.MemberGroups' });
 
-    $scope.openGroupsDictionarySettingManagement = function () {
-        var newBlade = {
-            id: 'settingDetailChild',
-            isApiSave: true,
-            currentEntityId: 'Customer.MemberGroups',
-            parentRefresh: function (data) { $scope.groups = data; },
-            controller: 'platformWebApp.settingDictionaryController',
-            template: '$(Platform)/Scripts/app/settings/blades/setting-dictionary.tpl.html'
-        };
-        bladeNavigationService.showBlade(newBlade, blade);
-    };
+            $scope.statuseSettings = settings.get({ id: 'Customer.ContactStatuses' }, function (data) {
+                if (blade.isNew) {
+                    blade.currentEntity.status = data.defaultValue;
+                    blade.origEntity.status = data.defaultValue;
+                }
+            });
 
-    $scope.fetchCustomerOrganizations = function($select) {
-        $scope.fetchOrganizations($select).then(function() {
-            $scope.customerOrganizations = angular.copy($scope.organizations);
-        });
-    }
+            blade.openStatusSettingManagement = function () {
```

Comment:
Status is property of a base Member type. Move as much as possible to base memberDetailController: setting loading and openStatusSettingManagement

Reply:
Yep, moved.